File: source/kCura.Relativity.Export.ConvertedToC#/Exporters/ImageExportInfo.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Exports
{
	public class ImageExportInfo
	{
		private string _fileName;
		private string _fileGuid;
		private Int32 _artifactID;
		private string _batesNumber;
		private string _tempLocation;
		private string _sourceLocation;
		private Nullable<Int32> _pageOffset;

		private bool _hasBeenDownloaded = false;
		public string FileName {
			get { return _fileName; }
			set { _fileName = value; }
		}

		public string FileGuid {
			get { return _fileGuid; }
			set { _fileGuid = value; }
		}

		public Int32 ArtifactID {
			get { return _artifactID; }
			set { _artifactID = value; }
		}

		public string BatesNumber {
			get { return _batesNumber; }
			set { _batesNumber = value; }
		}

		public string TempLocation {
			get { return _tempLocation; }
			set { _tempLocation = value; }
		}

		public string SourceLocation {
			get { return _sourceLocation; }
			set { _sourceLocation = value; }
		}

		public Nullable<Int32> PageOffset {
			get { return _pageOffset; }
			set { _pageOffset = value; }
		}

		public bool HasBeenCounted {
			get { return _hasBeenDownloaded; }
			set { _hasBeenDownloaded = value; }
		}

	}
}

File: source/kCura.Relativity.Export/Exporters/ImageExportInfo.vb
Namespace kCura.Relativity.Export.Exports
	Public Class ImageExportInfo
		Private _fileName As String
		Private _fileGuid As String
		Private _artifactID As Int32
		Private _batesNumber As String
		Private _tempLocation As String
		Private _sourceLocation As String
		Private _pageOffset As Nullable(Of Int32)
		Private _hasBeenDownloaded As Boolean = False

		Public Property FileName() As String
			Get
				Return _fileName
			End Get
			Set(ByVal value As String)
				_fileName = value
			End Set
		End Property

		Public Property FileGuid() As String
			Get
				Return _fileGuid
			End Get
			Set(ByVal value As String)
				_fileGuid = value
			End Set
		End Property

		Public Property ArtifactID() As Int32
			Get
				Return _artifactID
			End Get
			Set(ByVal value As Int32)
				_artifactID = value
			End Set
		End Property

		Public Property BatesNumber() As String
			Get
				Return _batesNumber
			End Get
			Set(ByVal value As String)
				_batesNumber = value
			End Set
		End Property

		Public Property TempLocation() As String
			Get
				Return _tempLocation
			End Get
			Set(ByVal value As String)
				_tempLocation = value
			End Set
		End Property

		Public Property SourceLocation() As String
			Get
				Return _sourceLocation
			End Get
			Set(ByVal value As String)
				_sourceLocation = value
			End Set
		End Property

		Public Property PageOffset() As Nullable(Of Int32)
			Get
				Return _pageOffset
			End Get
			Set(ByVal value As Nullable(Of Int32))
				_pageOffset = value
			End Set
		End Property

		Public Property HasBeenCounted() As Boolean
			Get
				Return _hasBeenDownloaded
			End Get
			Set(ByVal value As Boolean)
				_hasBeenDownloaded = value
			End Set
		End Property

	End Class
End Namespace